src/sentry/api/serializers/models/organization_member/expand/roles.py

```diff
@@ -57,11 +57,12 @@ def serialize(
         organization_role_list = [
             role for role in organization_roles.get_all() if not _is_retired_role_hidden(role, obj)
         ]
-        context["roles"] = serialize(
+        context["orgRoleList"] = serialize(
             organization_role_list,
             serializer=OrganizationRoleSerializer(),
             allowed_roles=self.allowed_roles,
         )
-        context["teamRoles"] = serialize(team_roles.get_all(), serializer=TeamRoleSerializer())
+        context["roles"] = context["orgRoleList"]
+        context["teamRoleList"] = serialize(team_roles.get_all(), serializer=TeamRoleSerializer())
```

Comment:
It's unclear if `teamRoles` are the list of options, or the list of team+roles that the user is in.

Reply:
The list is passed from the backend to the frontend, as it can be configured with custom roles in `server.py`.

We don't have somewhere on the frontend to store it, so it's either we do prop drilling or build a new hook/store. I've opted to following the existing pattern with organization roles.